test/rubygems/test_project_sanity.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require "rubygems/test_case"
+require "open3"
+
+class TestProjectSanity < Minitest::Test
+
+  def test_rake_package_builds_ok
+    skip unless File.exist?(File.expand_path("../../../Rakefile", __FILE__))
+
+    with_empty_pkg_folder do
+      output, status = Open3.capture2e("rake package")
+
+      assert_equal true, status.success?, <<~MSG.chomp
+        Expected `rake package` to work, but got errors:
+
+        ```
+        #{output}
+        ```
+
+        If you have added or removed files, make sure you run `rake update_manifest` to update the `Manifest.txt` accordingly
+      MSG
+    end
+  end
+
+  def test_manifest_is_up_to_date
+    skip unless File.exist?(File.expand_path("../../../Rakefile", __FILE__))
+
+    _, status = Open3.capture2e("rake check_manifest")
+
+    assert status.success?, "Expected Manifest.txt to be up to date, but it's not. Run `rake update_manifest` to sync it."
+  end
+
+  private
+
+  def with_empty_pkg_folder
+    if File.exist?("pkg")
+      FileUtils.cp_r "pkg", "tmp"
```

Comment:
Any specific reason why these kind of methods don't have parenthesis but others do?

Reply:
No. Do you want me to add parentheses or something?

Reply:
@deivid-rodriguez I think that's the preferred style now, though the current rubocop.yml doesn't seem to enforce it. But, I will defer to @bronzdoc and the rest of the rubygems community. :)

Reply:
Yeah, we talked about it in the past, but the preferred style is not consistent. We prefer no parenthesis for many methods: most methods in `Kernel`, most Minitest assertions, and more. So I don't usually care much about it.

Reply:
> We prefer no parenthesis for many methods: most methods in Kernel, most Minitest assertions

I would not put parenthesis in those either since is idiomatic and the lack of a specific receiver make it read more like english. (My personal opinion)

> No. Do you want me to add parentheses or something?

If is not much of a problem for you. Yes, for the ones with a specific receiver.  🙇

Reply:
Not at all, and I agree with your opinion. I'll give it another try at enforcing that style through rubocop.

Reply:
Thought I had commented here... Anyways, I did a quick try to enforce this through rubocop and I don't think it's worth it since even with a configuration very flexible in terms of exceptions, we still have more than 7K offenses.

Anyways, fixed the specific styling here.